src/test/java/com/exasol/sql/expression/rendering/ExasolUdfFunctionRenderingTest.java

```diff
@@ -0,0 +1,32 @@
+package com.exasol.sql.expression.rendering;
+
+import static com.exasol.hamcrest.SqlFragmentRenderResultMatcher.rendersTo;
+import static com.exasol.sql.expression.ExpressionTerm.column;
+import static com.exasol.sql.expression.ExpressionTerm.integerLiteral;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+import com.exasol.sql.ColumnsDefinition;
+import com.exasol.sql.StatementFactory;
+import com.exasol.sql.dql.select.Select;
+
+class ExasolUdfFunctionRenderingTest {
+    @Test
+    void testUdfFunction() {
+        final Select select = StatementFactory.getInstance().select().udfFunction("my_average", column("x"));
```

Comment:
`udfFunction` returns `Select` reference, so please use fluent programming.

Reply:
Again, I can't. If I append it now, I'll get a FromClause object, and I don't need it. I need a Select.

Believe me, I use fluent programming in every possible place...